core/src/main/java/feast/core/config/FeatureStreamConfig.java

```diff
@@ -139,13 +146,43 @@ public Source getDefaultSource(FeastProperties feastProperties) {
       case KAFKA:
         String bootstrapServers = streamProperties.getOptions().getBootstrapServers();
         String topicName = streamProperties.getOptions().getTopic();
+        Map<String, Object> map = new HashMap<>();
```

Comment:
This one resolved incorrectly, topic creation was moved to dedicated Bean

Reply:
Not entirely sure about this one. I did a diff with master and removed the code that was extraneous. Mind  having a look to see if i did  this correctly